dev/breeze/doc/BREEZE.md

```diff
@@ -0,0 +1,108 @@
+<!--
```

Comment:
I think this one should be removed completely ?

Reply:
Thank you, I did this time.